hw/hw1/ex_02.py

```diff
@@ -0,0 +1,27 @@
+# Задача 4: Петя, Катя и Сережа делают из 
+# бумаги журавликов. Вместе они сделали S журавликов.
+#  Сколько журавликов сделал каждый ребенок, если известно, 
+#  что Петя и Сережа сделали одинаковое количество журавликов, 
+#  а Катя сделала в два раза больше журавликов, 
+#  чем Петя и Сережа вместе?
+
+# *Пример:*
+# 6 -> 1  4  1
+# 24 -> 4  16  4
+# 60 -> 10  40  10
+
+#result = x + x + (x * 4)
+print("введите общее количество журавликов: ")
+result = int(input()) 
+
+count = 1
+while(result != count*6):
+    count += 1
+if result == count*6:
+    print("Петя и Сережа сделали по ", count, " журавлика(ов), а Катя ", count*4, " журавлика(ов)") 
```

Comment:
Не забывайте проводить код к виду PEP8.
```suggestion
    print(f"Петя и Сережа сделали по {count} журавлика(ов), а Катя {count * 4} журавлика(ов)") 
```